todo_app/app.py

```diff
@@ -22,17 +26,28 @@ def create_app(db = None):
     app = Flask(__name__)
     app.config.from_object('todo_app.flask_config.Config')
     app.config['LOGIN_DISABLED'] = os.getenv('LOGIN_DISABLED')
-
+    app.config['LOGGLY_TOKEN'] = os.getenv('LOGGLY_TOKEN')
+    #logging.basicConfig(level = os.getenv('LOG_LEVEL')) 
+    app.config['LOG_LEVEL'] = os.getenv('LOG_LEVEL') 
```

Comment:
Would it be worth adding a default value for this? e.g.
```suggestion
    app.config['LOG_LEVEL'] = os.getenv('LOG_LEVEL', 'INFO')
```

Since this isn't sensitive and we're likely to agree a default level for most servers, this might be quite a reasonable way to cut down how many environment variables we need to set elsewhere